homeassistant/components/vizio/config_flow.py

```diff
@@ -180,8 +180,11 @@ def __init__(self) -> None:
                 if entry.data[CONF_NAME] != import_config[CONF_NAME]:
                     updated_name[CONF_NAME] = import_config[CONF_NAME]
 
-                if entry.data[CONF_VOLUME_STEP] != import_config[CONF_VOLUME_STEP]:
-                    updated_options[CONF_VOLUME_STEP] = import_config[CONF_VOLUME_STEP]
+                import_volume_step = import_config.get(
```

Comment:
The imported config already has a default for the volume step item.

Reply:
That's a good point, the original bug had to do with invalid key but that error must have been on the entry.data not on the imported config. Will submit a fix